src/operator/sequence_mask.cc

```diff
@@ -43,20 +43,79 @@ Operator *SequenceMaskProp::CreateOperatorEx(Context ctx,
 DMLC_REGISTER_PARAMETER(SequenceMaskParam);
 
 MXNET_REGISTER_OP_PROPERTY(SequenceMask, SequenceMaskProp)
-    .describe(
-"Sets all elements outside the sequence to a constant value. Takes an n-dimensional tensor of the "
-"form [max sequence length, batchsize, other dims] and returns a tensor of the same "
-"shape. This operator takes an optional input tensor sequence_length of positive ints of "
-"dimension [batchsize] when the sequence_length option is set to true. This allows the "
-"operator to handle variable-length sequences. If sequence_length is false, then each "
-"example in the batch is assumed to have the max sequence length, and this operator becomes "
-"the identity operator."
-)
+    .describe(R"code(Sets all elements outside the sequence to a constant value.
+
+This function takes an n-dimensional input array of the form [max sequence length, batch size, other dims]
```

Comment:
maybe better to indicate some how that "max sequence length" refers to a single quantity (it does, correct?). I suggest just using underscores:

 [max_sequence_length, batch_size, other dims]

Now for "other dims" -- what does that mean? it's not obvious form the context.